dags/process_md_1c_to_pg/libs/transform.py

```diff
@@ -0,0 +1,55 @@
+import csv
+import json
+import os
+import uuid
+import pandas as pd
+import logging
+
+from .mapping import MdFieldsMap
+
+def transform_data(in_fp: str, out_dp: str, src_map: dict, dest_map: dict, file_key: str) -> str:
+    df = pd.read_excel(in_fp, dtype=str)
+    df.rename(columns=src_map, inplace=True)
+    dest_columns = list(dest_map.keys())
+    df = df[df.columns.intersection(dest_columns)]
+    for col in dest_columns:
+        if col not in df.columns:
+            df[col] = None
+    df = df[dest_columns]
+    export_fp = os.path.join(out_dp, f"{uuid.uuid4().hex}_{file_key}.csv")
+    df.to_csv(export_fp,
+              index=False,
+              encoding='utf-8',
+              sep=',',
+              quotechar='"',
+              quoting=csv.QUOTE_MINIMAL)
```

Comment:
Метод `to_csv()` датафрейма по умолчанию не гарантирует сохранение порядка колонок; соответственно, могут возникнуть проблемы при дальнейшем импорте в postgresql. Поэтому необходимо в метод явно передать список колонок в нужном порядке при помощи аргумента `columns`.
```suggestion
    df.to_csv(export_fp,
              index=False,
              encoding='utf-8',
              sep=',',
              quotechar='"',
              quoting=csv.QUOTE_MINIMAL,
              columns=dest_columns)
```